airbyte-integrations/connectors/destination-duckdb/destination_duckdb/destination.py

```diff
@@ -150,11 +158,16 @@ def check(self, logger: AirbyteLogger, config: Mapping[str, Any]) -> AirbyteConn
         :return: AirbyteConnectionStatus indicating a Success or Failure
         """
         try:
-            # parse the destination path
-            param_path = config.get("destination_path")
-            path = self._get_destination_path(param_path)
+            path = config.get("destination_path")
```

Comment:
We should probably check whether the path is motherduck and fail the check otherwise if in a cloud environment

Reply:
Makes sense. When I wrote this I didn't know how to make that condition check, but you shared a code snippet elsewhere which we can probably reuse here.

Reply:
Logged as follow-up:

- https://github.com/airbytehq/airbyte/issues/30590